src/VSCodeExtension/syntaxes/qsharp.tmLanguage.json

```diff
@@ -48,7 +48,7 @@
         },
         {
           "name": "keyword.other.qsharp",
-          "match": "\\b(new|not|and|or|w\/)\\b"
+          "match": "\\b(new|not|and|or|w\/|size)\\b"
```

Comment:
The tricky thing about `size` is that it's not a true keyword like `not`. It acts like a keyword when it occurs in a specific part of an array literal, but it's still a valid identifier:

```qsharp
let size = 123;
let xs = [size, size = 2]; // first size is a variable, second size is a "keyword"
```

So I am not sure if we want all instances of `size` highlighted.

Reply:
Hmm, let me see if I can make something like "a comma, optional whitespace, size, optional whitespace, equality sign" work  without also highlighting the comma and the equality...